docs/integrations/app-integrations/mediawiki.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: MediaWiki
+sidebarTitle: MediaWiki
+---
+
+This is the implementation of the MediaWiki data handler for MindsDB.
+
+[MediaWiki](https://www.mediawiki.org/wiki/MediaWiki) is a free server-based wiki software, licensed under the GNU General Public License (GPL). It's designed to serve a website that gets millions of hits per day.
+
+## Implementation
```

Comment:
To keep app doc pages consistent, let's rename this chapter to `## Connection` and move the `CREATE DATABASE` statement to this chapter.